modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -0,0 +1,204 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-top-aware_{context}"]
+= vSphere Topology Awareness
+
+{product-title} provides the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple data centers and clusters, thus helping to avoid a single-point-of-failure. 
+
+This is accomplished by defining zone and region categories in vCenter, and then assigning these categories to different failure domains, such as a compute cluster, host, or data center, by creating tags for these zone and region categories. After you have created the appropriate categories, and assigned tags to vCenter objects, you can create additional machinesets that create virtual machines (VMs) that are responsible for scheduling pods in those failure domains.
+
+.Procedure
+. In the VMware vCenter vSphere client GUI, define appropriate zone and region catagories and tags.
++
+Zones and regions in the vSphere environment are typically defined by creating categories called `openshift-zone` and `openshift-region`.
+Usually, category names can be any valid string, but {product-title} recommends use of `openshift-zone` and `openshift-region`
+categories. 
++
+The following example defines two failure domains with one region and two zones:
```

Comment:
Should we say "The following table" since it's a table?

Reply:
I want to emphasis that this is example data--you can set this up many possible different ways, so I'm going to leave it.